tests/client/amo/testApi.js

```diff
@@ -204,20 +216,21 @@ describe('amo.api', () => {
         .expects('callApi')
         .returns(Promise.resolve({ results: [] }));
 
-      return getLatestUserReview({ user: 123, addon: 321 })
+      return getLatestUserReview({ user: 123, addon: 321, version: 456 })
         .then((review) => {
           assert.strictEqual(review, null);
         });
     });
 
-    it('requires a user and addon', () => {
+    it('requires a user, addon, and version', () => {
```

Comment:
Yeah, true. Those kind of tests are a bit annoying and hard to get right but I'll add them.